### ISSUE TITLE
Plugin doesn't work what so ever

### ISSUE THREAD

**Issue by Seek0999** [Solaris9/erela.js-spotify#30]
## All values have been pre-checked and non returned null or an invalid value. The null values originated from inside of the plugin

# Error when trying to play a playlist:
```
{
  message: 'The "url" argument must be of type string. Received null',
  severity: 'COMMON'
}
```

# Error when trying to play a author:
```
{ severity: 'COMMON', message: 'Unknown file format.' }
```

# Error when trying to play a album:
```
{
  message: 'The "url" argument must be of type string. Received null',
  severity: 'COMMON'
}
```

### Only thing that works is playlist singular tracks, theres something messed up with the library so i'd appreciate it if you could take a look. Thanks!

**Comment by TheGamer3514** [comment]
same issue here

**Comment by Seek0999** [comment]
> same issue here

Yeah im guessing the plugin is deprecated as no one is maintaining it, I'd recommend the plugin called better-erela.js-spotify as its actively maintained and also works.
